crates/bevy_core_pipeline/src/skybox/mod.rs

```diff
@@ -90,6 +92,17 @@ pub struct Skybox {
     /// After applying this multiplier to the image samples, the resulting values should
     /// be in units of [cd/m^2](https://en.wikipedia.org/wiki/Candela_per_square_metre).
     pub brightness: f32,
+    pub rotation: Quat,
```

Comment:
Docs would be nice here.

Reply:
Documentation has been added to explain the rotation and its appropriate usage.